### PR TITLE
update ray version in default template and code references

### PR DESCRIPTION
**Pull request by KPostOffice**
## Checks
- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [x] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: tests/unit_test.py [hunk 1]
```diff
@@ -2294,9 +2294,9 @@ def test_export_env():
 
 # Make sure to always keep this function last
 def test_cleanup():
-    os.remove("unit-test-cluster.yaml")
-    os.remove("prio-test-cluster.yaml")
-    os.remove("unit-test-default-cluster.yaml")
+    # os.remove("unit-test-cluster.yaml")
+    # os.remove("prio-test-cluster.yaml")
+    # os.remove("unit-test-default-cluster.yaml")
```
Comment (Maxusmusti) [inline tests/unit_test.py:2299]: Assuming you want to uncomment these?

**Review by Maxusmusti — CHANGES_REQUESTED** [review]

File: tests/unit_test.py (hunk 1 above)
Comment (KPostOffice) [reply tests/unit_test.py:2299]: Oops, yes thanks

**Review by Maxusmusti — COMMENTED** [review]
/lgtm

**Review by anishasthana — COMMENTED** [review]
/lgtm

File: custom-nb-image/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM  quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8-4c8f26e
+FROM  quay.io/opendatahub/notebooks:jupyter-minimal-ubi9-python-3.9-2023a_20230302
 # Install: torch (v1.12), ray (v2.1.0) and others
```
Comment (dimakis) [inline custom-nb-image/Dockerfile:16]: nit:
can we update/  change this to leave out the versions? I think its pretty self explanatory anyway

**Review by dimakis — COMMENTED** [review]
lgtm

File: custom-nb-image/Dockerfile (hunk 2 above)
Comment (Maxusmusti) [reply custom-nb-image/Dockerfile:16]: I'll wait until after that comment is removed/updated to approve, @KPostOffice

**Review by Maxusmusti — APPROVED** [review]
LGTM

**Pull request merged**